app/src/helpers/requests/lloydGeorgeSearchResult.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+type Args = {
+    nhsNumber: string;
+    baseUrl: string;
+};
+
+export type LloydGeorgeStitchResult = {
+    number_of_files: number;
+    total_file_size_in_byte: number;
+    last_updated: string;
+    presign_url: string;
+};
+
+async function getLloydGeorgeRecord({
+    nhsNumber,
+    baseUrl,
+}: Args): Promise<LloydGeorgeStitchResult> {
+    const gatewayUrl = baseUrl + '/LloydGeorgeStitch';
+
+    const { data } = await axios.get(gatewayUrl, {
+        headers: {
+            'Content-Type': 'application/json',
```

Comment:
This will need to use the new "useBaseAPIHeaders" which will contain the auth data, it's merged into main now so rebase and add that hook in similar to how baseUrl hook works. Find another request example :)